source/user/FAQ.md

```diff
@@ -34,9 +34,9 @@ Yes. Those restrictions do not control how you use the code, merely how you trea
 
 For most users, yes.
 
-Most people do not use an operating system. Instead, they use applications, as these are what make a computer useful. GhostBSD is designed to provide a desktop and full-featured environment for applications. It supports various web browsers, office suites, email readers, graphics programs, programming environments, network servers, and just about everything else you might want. Most of these applications can be built using the FreeBSD Ports Collection.
+Most people do not use an operating system. Instead, they use applications that make a computer useful. GhostBSD is designed to provide a desktop and full-featured environment for applications. It supports various web browsers, office suites, email readers, graphics programs, programming environments, network servers, and everything else you might want. Most of these applications can be built using the FreeBSD Ports Collection.
 
-Suppose you need to use an application that is only available on one operating system. In that case, you cannot easily replace that operating system (this situation is referred to as vendor lock-in). However, the chances are that there is a very similar application on GhostBSD.
+Suppose you need to use an application only available on an operating system will be hard to replace that operating system (this situation is called vendor lock-in). However, the chances are that there is a very similar application on GhostBSD.
```

Comment:
The first sentence sounds weird to me.


```suggestion
If you need to use an application only available on a specific operating system, it may be hard to replace that operating system (this situation is called vendor lock-in). However, chances are that there is a very similar alternative application on GhostBSD.
```